nearcore/src/config.rs

```diff
@@ -1257,21 +1257,28 @@ impl From<NodeKeyFile> for KeyFile {
     }
 }
 
-pub fn load_config(dir: &Path, genesis_validation: GenesisValidationMode) -> NearConfig {
-    let config = Config::from_file(&dir.join(CONFIG_FILENAME)).unwrap();
+pub fn load_config(
+    dir: &Path,
+    genesis_validation: GenesisValidationMode,
+) -> anyhow::Result<NearConfig> {
```

Comment:
I would rather use, `Result<NearConfig, anyhow::Error>`; This should be much easier to understand, the result type is from std, while the only thing that's different is the error type.

Reply:
done